data/movements/scripts/walkback.lua

```diff
@@ -1,6 +1,6 @@
 function onStepIn(creature, item, position, fromPosition)
 	if item.uid > 0 and item.uid <= 65535 then
-		creature:teleportTo(fromPosition, false)
+		return false
 	end
 	return true
```

Comment:
But it is pushed as an `uint16_t`, isn't it?

Anyway it can be `return item.uid == 0 or item.uid > 65535`